spotbugs/src/main/java/edu/umd/cs/findbugs/FindBugs2.java

```diff
@@ -742,8 +742,7 @@ private void buildReferencedClassSet() throws InterruptedException {
         }
         Set<String> referencedPackageSet = new HashSet<>();
 
-        LinkedList<ClassDescriptor> workList = new LinkedList<>();
-        workList.addAll(appClassList);
+        LinkedList<ClassDescriptor> workList = new LinkedList<>(appClassList);
```

Comment:
While this is being cleaned up, you could probably also change the variable to use the interface type, `List`, instead of the concrete type.

Reply:
Yes indeed it would be much better. But i would like to do such things by regrouping modification by type.